tests/test_resolve.py

```diff
@@ -196,20 +222,72 @@ def testResolveStarImportBuiltin(self):
     def testResolveStarImportSystem(self):
         imp = parsepy.ImportStatement("f", is_from=True, is_star=True,
                                       source="/system/f.py")
-        r = resolve.Resolver(self.path, "x.py")
+        r = self.make_resolver("x.py", "x")
         f = r.resolve_import(imp)
         self.assertEqual(f.path, "/system/f.py")
         self.assertEqual(f.module_name, "f")
 
     def testResolvePyiFile(self):
         imp = parsepy.ImportStatement("x")
-        r = resolve.Resolver(self.path, "b.py")
+        r = self.make_resolver("b.py", "b")
         f = r.resolve_import(imp)
         self.assertTrue(isinstance(f, resolve.Local))
         self.assertEqual(f.fs, self.pyi_fs)
         self.assertEqual(f.path, "x.pyi")
         self.assertEqual(f.module_name, "x")
 
+    def testResolveSystemRelative(self):
+        with utils.Tempdir() as d:
+            os_fs = fs.OSFileSystem(d.path)
+            fspath = [os_fs]
+            py_file = d.create_file("foo/x.py")
+            py_file = d.create_file("foo/y.py")
```

Comment:
flake8: ./tests/test_resolve.py:244:13: F841 local variable 'py_file' is assigned to but never used.